### PR TITLE
Upgrade to solve 2.0

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: spec/unit/berkshelf/berksfile_spec.rb [hunk 1]
```diff
@@ -238,7 +238,8 @@
   describe '#cookbooks' do
     it 'raises an exception if a cookbook is not installed' do
       subject.add_dependency('bacon', nil)
-      expect { subject.cookbooks }.to raise_error
+      subject.cookbooks
```
Comment (reset) [inline spec/unit/berkshelf/berksfile_spec.rb]: Pretty sure you didn't mean to include this line ;)

**Comment by martinb3** [comment]
Major kudos on this one. This has fixed a _ton_ of issues for us.

**Comment by jkeiser** [comment]
@reset fixed the issue :) Also upgraded the rake dependency, which doesn't seem to cause any issues.

**Comment by reset** [comment]
Looks great, thanks @jkeiser

**Pull request merged**